EnglishTranslation-master/ETC.tsv

```diff
@@ -27925,21 +27925,21 @@ ETC_20170605_027919	A package that contains a My Wedding Costume. Right-click to
 ETC_20170605_027920	Soul Crystal (4 Medals)	소울 크리스탈 (메달 4개)				
 ETC_20170605_027921	Using this Soul Crystal will resurrect you in the same position you died.{nl} Can be exchanged for 4 medals in the Exchange Shop.	전투 불능 시 소울 크리스탈을 사용하여 제자리에서 부활할 수 있습니다.{nl} 리사이클샵에서 메달 4개로 교환 가능합니다.				
 ETC_20170605_027922	Attribute Points 1,000 Ticket	특성 포인트 1,000				
-ETC_20170605_027923	Will increase attribute points by 1,000. Right-click to use. 	사용시 특성 포인트가 1,000 증가합니다. 마우스 오른쪽 클릭으로 사용합니다.				
+ETC_20170605_027923	Increases your Attribute Points by 1,000. Right-click to use. 	사용시 특성 포인트가 1,000 증가합니다. 마우스 오른쪽 클릭으로 사용합니다.				
 ETC_20170605_027924	Shining Stat Reset Potion 	빛나는 스탯 초기화 포션				
 ETC_20170605_027925	Shining Skill Reset Potion 	빛나는 스킬 초기화 포션				
 ETC_20170605_027926	Attribute Points Box (Rank S) 	특성 포인트 상자 S급				
-ETC_20170605_027927	A box that contains 10 Attribute Points 1,000 Tickets. Right-click to obtain. 	사용시 1,000 특성 포인트가 10장 들어있는 상자입니다. 마우스 오른쪽 버튼 클릭으로 사용하면 아이템이 지급됩니다.				
+ETC_20170605_027927	A box that contains ten Attribute Points 1,000 Tickets. Right-click to obtain. 	사용시 1,000 특성 포인트가 10장 들어있는 상자입니다. 마우스 오른쪽 버튼 클릭으로 사용하면 아이템이 지급됩니다.				
```

Comment:
why change this? the original Korean translation used the number.

Reply:
Perhaps it could be misread as `A box that contains 10 attribute points` to the skimming reader, but even with the change it still has the same meaning, It should be alright.

Reply:
The sentence could use a touch up. Like @sunhwapark mentioned, it could be misread. How about renaming the actual item to "Attribute Points (1,000) Ticket"? - Reads much better and less confusing instead of 1,000 tickets.

`A box that contains 10 Attribute Points (1,000) Tickets`